### PR TITLE
Add interactive list with add & remove feature  

### PR DESCRIPTION
**Pull request by imranhossainemi**
**Update project with add & remove feature**

- update index.js file 
- create list.js file 
- update list.js file

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,37 +1,22 @@
...
-    <div class="ver"><i class="fa-sharp fa-solid fa-ellipsis-vertical"></i></div>
-    <div class="trash"><i class="fa-sharp fa-solid fa-trash"></i></div>
-    `;
-    doList.appendChild(li);
-  }
-};
-listItem();
+list.addTask();
+list.showTask();
+// interactive();
```
Comment (Ismailco) [inline src/index.js]: - [ ] [OPTIONAL] Kindly consider removing old code from your files, avoiding comments in your code to make it easier to read

File: src/list.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+let listArr = JSON.parse(localStorage.getItem('task')) || [];
+export default class List {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  addTask = () => {
```
Comment (Ismailco) [inline src/list.js:9]: - [ ] Kindly make sure that new tasks use the next index in the list, as for now, all the tasks have `1` as the index. one way to fix this is by setting the index to the length of the array in the local storage, this way the new task will have the proper index
- [ ] Also make sure when deleting a task the indexes of the other task should be changed

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @imranhossainemi, 

Good job so far 👍🏼 
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @imranhossainemi , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**